pkg/controller/logstorage/kubecontrollers/es_kube_controllers.go

```diff
@@ -111,13 +116,27 @@ func Add(mgr manager.Manager, opts options.AddOptions) error {
 	if err = utils.AddTigeraStatusWatch(c, initializer.TigeraStatusLogStorageKubeController); err != nil {
 		return fmt.Errorf("logstorage-controller failed to watch logstorage Tigerastatus: %w", err)
 	}
+	if opts.MultiTenant {
+		if err = c.Watch(&source.Kind{Type: &operatorv1.Tenant{}}, &handler.EnqueueRequestForObject{}); err != nil {
+			return fmt.Errorf("log-storage-kubecontrollers failed to watch Tenant resource: %w", err)
+		}
+	}
+
+	// The namespace(s) we need to monitor depend upon what tenancy mode we're running in.
+	// For single-tenant, everything is installed in the calico-system namespace.
+	// Make a helper for determining which namespaces to use based on tenancy mode.
+	esGatewayHelper := utils.NewNamespaceHelper(opts.MultiTenant, render.ElasticsearchNamespace, "")
```

Comment:
Wouldn't it be better to do:
```
	esgwNamespace := utils.NewNamespaceHelper(opts.MultiTenant, render.ElasticsearchNamespace, "").InstallNamespace()
	kubecontrollersNamespace := utils.NewNamespaceHelper(opts.MultiTenant, common.CalicoNamespace, "").InstallNamespace()

```

And move the lines to right above the place they are used? Imo the code is good enough to do without the comment, but I don't mind keeping it either.